### PR TITLE
test: add network interception fail request invalidation tests

### PR DESCRIPTION
**Pull request by thiagowfx**

### CODE REVIEW

**Review by sadym-chromium — APPROVED** [review]

File: webdriver/tests/bidi/network/fail_request/invalid.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    network_events = await setup_network_test(events=[
+        "network.responseCompleted",
+    ])
+
+    text_url = url(PAGE_EMPTY_TEXT)
+    on_response_completed = wait_for_event("network.responseCompleted")
+    await fetch(text_url)
+    await on_response_completed
+
+    request = network_events["network.responseCompleted"][0]["request"]["request"]
```
Comment (juliandescottes) [inline webdriver/tests/bidi/network/fail_request/invalid.py:32]:
```suggestion
    response_completed_event = await on_response_completed

    request = response_completed_event["request"]["request"]
```

We could simplify a bit and avoid reading the array.

File: tools/webdriver/webdriver/bidi/modules/network.py [hunk 2]
```diff
@@ -61,3 +61,8 @@ def _add_intercept(self, result: Mapping[str, Any]) -> Any:
     def remove_intercept(self, intercept: str) -> Mapping[str, Any]:
         params: MutableMapping[str, Any] = {"intercept": intercept}
         return params
+
+    @command
+    def fail_request(self, request: str) -> Mapping[str, Any]:
+        params: MutableMapping[str, Any] = {"request": request}
+        return params
```
Comment (juliandescottes) [inline tools/webdriver/webdriver/bidi/modules/network.py:68]:
```suggestion
    def fail_request(self, request: str) -> Mapping[str, Any]:
        params: MutableMapping[str, Any] = {"request": request}
        return params

    @command
    def remove_intercept(self, intercept: str) -> Mapping[str, Any]:
        params: MutableMapping[str, Any] = {"intercept": intercept}
        return params
```

nit: maybe keep the commands alphabetically sorted.

**Review by juliandescottes — APPROVED** [review]
Thanks for the new tests! LGTM

File: tools/webdriver/webdriver/bidi/modules/network.py (hunk 2 above)
Comment (thiagowfx) [reply tools/webdriver/webdriver/bidi/modules/network.py:68]: Done

File: webdriver/tests/bidi/network/fail_request/invalid.py (hunk 1 above)
Comment (thiagowfx) [reply webdriver/tests/bidi/network/fail_request/invalid.py:32]: Done

**Pull request merged**